gwak/train/dataloader.py

```diff
@@ -326,9 +377,21 @@ def inject(self, batch, waveforms):
         # calculate psds
         psds = spectral_density(psd_data.double())
 
-        # inject into data and whiten
-        injected = batch + waveforms
-
+        # Waveform padding
+        inj_len = waveforms.shape[-1]
+        window_len = splits[1]
+        half = int((window_len - inj_len)/2)
+
+        first_half, second_half = half, window_len - half - inj_len
+        
+        waveforms = F.pad(
+            input=waveforms, 
+            pad=(first_half, second_half), 
+            mode='constant', 
+            value=0
+        )
+        
+        injected = batch + waveforms * 100
```

Comment:
Where does the `* 100` come from?

Reply:
This is for rescaling the distance of bbh at a closer distance. In the future this should be using SNRResclaer to inject BBH at different SNR.